src/osg/Drawable.cpp

```diff
@@ -226,16 +227,15 @@ Drawable::Drawable()
     _useDisplayList = false;
 #endif
 
-#if 0
-    _supportsVertexBufferObjects = false;
-    //_useVertexBufferObjects = false;
-    _useVertexBufferObjects = false;
-#else
-    _supportsVertexBufferObjects = true;
-    _useVertexBufferObjects = true;
-#endif
+    _supportsVertexBufferObjects = true; // OSG 3.6
+    _useVertexBufferObjects = false;  // MAK OSG 3.4 (OSG 3.6 = true)
```

Comment:
It looks like we set the default to be on only for Oculus but maybe we can turn it on for everything now and get rid of the s_useVertexBufferObjectsDefault and the setter function for that. I guess lets leave it off for the initial 3.6 merge but put in a defect to ontime for trying to turn it on all the time and see if it works.